src/functions.php

```diff
@@ -369,6 +369,15 @@ function upload(string $xml, $config)
     $fritz->mergeClientOptions($options['http'] ?? []);
     $fritz->login();
 
+    if (!phoneNumberAttributesSet($xmlNewPhonebook)) {
```

Comment:
I think this distinction needs to mentioned in the readme. Essentially there are 2 differenz ways to achieve the desired behaviour.

Reply:
OK, added feature & backwards compatible hint to Readme.md